assets/js/googlesitekit-data.js

```diff
@@ -1,10 +1,15 @@
 /**
  * External dependencies
  */
-import Data from 'assets/js/googlesitekit/data';
+import SiteKitRegistry, * as DataFunctions from 'assets/js/googlesitekit/data';
 
 if ( typeof global.googlesitekit === 'undefined' ) {
 	global.googlesitekit = {};
 }
 
-global.googlesitekit.data = Data;
+if ( typeof global.googlesitekit.data === 'undefined' ) {
+	global.googlesitekit.data = { ...DataFunctions, SiteKitRegistry };
```

Comment:
For things like https://github.com/google/site-kit-wp/blob/296fb1a7b2ed9fba6dd07b05e6ebba5f9b1132f7/assets/js/googlesitekit/site/index.test.js#L14, I thought I'd use `googlesitekit-data` as the "data" module source-of-truth instead of `@wordpress/data`. I figured it could get confusing if we start to use/mix both. It might make it a bit unclear as to when we use which.

I can rely on `@wordpress/data` instead, I don't really have a preference. But this just keep our JSR data functions isolated to `googlesitekit-data`. It means we might be able to limit the exports there too (and reduce file size) in the future via dead code removal, but that's not a priority 😄

Reply:
I'm not sure I follow. Indeed we should rely on `googlesitekit-data` as the source of truth for the store functions, but those are already present on our own registry that is exported (`SiteKitRegistry`). For other functions that we'd only be using for testing anyway we can preferably still rely on `@wordpress/data`, especially since these are also what we use for the same purpose in the production code (e.g. we will likely never use `wp.data.createRegistry` outside of this module and tests, as that should be our registry where all stores go).